### DISCUSSION TITLE
How to train multiple subjects in Dreambooth

### DISCUSSION THREAD

**Discussion by jjpb44** [TheLastBen/fast-stable-diffusion#1258] [Q&A]
I used to train models with multiple subjects by simply renaming the instance images differently (YYY (1).png, YYY (2).png...etc and ZZZ (1).png, ZZZ (1).png...etc) Is this still the method for training multiple subjects? Can I leave the captions empty or is it used to differentiate the subjects?

**Comment by nawnie** [comment]
still works keep text encoder at a low lr
